web/src/components/install/index.tsx

```diff
@@ -10,6 +10,10 @@ import { IoDownload } from 'react-icons/io5';
 export function InstallButton() {
   const [prompt, promptToInstall] = useAddToHomescreenPrompt();
 
+  if (!prompt) {
+    return null; 
```

Comment:
you shouldnt return a null in a component. please return <></>

Reply:
erased